src/lib/axes.typ

```diff
@@ -55,6 +55,8 @@
 
 // Format a tick value
 #let format-tick-value(value, tic-options) = {
+  if value == -0 { value = 0 }
```

Comment:
Typst doesn't have negative zeros?

Reply:
This code worked, though. Otherwise, we get negative zeros when doing the string conversion.

Reply:
Typst actually has negative zero floats, so this test makes sense.